AntennaTracker/control_auto.cpp

```diff
@@ -32,23 +35,32 @@ void Tracker::update_auto(void)
         update_yaw_servo(bf_yaw);
     }
 }
-void Tracker::calc_angle_error( float pitch, float yaw)
+void Tracker::calc_angle_error( float pitch, float yaw, bool direction_reversed)
 {
-	// Pitch angle error in centidegrees
-	// Positive error means the target is above current pitch
+    // Pitch angle error in centidegrees
+    // Positive error means the target is above current pitch
     // Negative error means the target is below current pitch
     float ahrs_pitch = ahrs.pitch_sensor;
     nav_status.angle_error_pitch = pitch - ahrs_pitch;
 
     // Yaw angle error in centidegrees
-	// Positive error means the target is right of current yaw
+    // Positive error means the target is right of current yaw
     // Negative error means the target is left of current yaw
     int32_t ahrs_yaw_cd = wrap_180_cd(ahrs.yaw_sensor);
     nav_status.angle_error_yaw = wrap_180_cd(yaw - ahrs_yaw_cd);
+    if(direction_reversed == true){
```

Comment:
small formatting issue, normally we have a space between the if and the bracket.  Also another space between the closing bracket and the squirly bracket.  So realy it should be:
     if (direction_reversed == true) {
Also I'd say no need for the "== true" part.  It doesn't make a functional difference but generally there's no need for this.